src/util.js

```diff
@@ -27,10 +27,15 @@ export function convertToQueryParams(o){
       case 'brandId':
       case 'categoryId':
       case 'facetBy':
-        let x = _.map(value, function(value){
-          return attr + '=' + value;
-        });
-        return x.join('&');
+        if (_.isArray(value)) {
+          let x = _.map(value, function(value){
```

Comment:
Let's not use lodash for trivial native methods. This was already pointed out [here](https://github.com/indix/indix-api-nodejs/pull/11#discussion_r107118419) and [here](https://github.com/indix/indix-api-nodejs/pull/11#discussion_r107118663). I'd recommend going over the review comments once again on #11 carefully. Also please avoid the use of single-character variable names like `x`; variable names should be self-explanatory.